react/modules/deprecated.tsx

```diff
@@ -14,19 +14,12 @@
  * */
 import React, { Component, ComponentType } from 'react'
 
-interface UseNewProps {
-  submitFilterLable: string
-  newFilterLable: string
-}
-
-interface UseNewComponent {
-  old: string
-  new: string
-}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
should we disable rules?

Reply:
in this case yes, because there is no way to know which type of prop is going to be passed.